src/translator/text-based-field.ts

```diff
@@ -75,7 +75,10 @@ export namespace TextBasedField {
         const validationObject = createValidationObject(element);
         const textBasedFieldSchema = {
             [element.id]: {
-                label: element.label,
+                label: {
+                    mainLabel: element.label,
+                    subLabel: element.description || "",
```

Comment:
add subLabel only if there is a value

```suggestion
                     ...(element.description && {
                        subLabel: element.description,
                    }),
```